.build/jenkins-deployment.yaml

```diff
@@ -0,0 +1,112 @@
+data:
+  1-system-message.yaml: |
+      tool:
+        jdk:
+          installations:
+            - name: jdk_11_latest
+              home: "/usr/bin/jdk_11/bin/java"
+            - name: jdk_17_latest
+              home: "/usr/bin/jdk_17/bin/java"
+---  
+apiVersion: jenkins.io/v1alpha2
+kind: Jenkins
+metadata:
+  name: example
+spec:
+  configurationAsCode:
+    configurations:
+    - name: jenkins-operator-user-configuration
+    secret:
+      name: ""
+  groovyScripts:
+    configurations: []
+    secret:
+      name: ""
+  service:
+    type: ClusterIP
+    port: 8080
+  jenkinsAPISettings:
+    authorizationStrategy: createUser
+  master:
+    plugins:
+    - name: ws-cleanup
+      version: "0.45"
+    - name: javadoc
+      version: "243.vb_b_503b_b_45537"
+    - name: description-setter
+      version: "1.10"
+    - name: throttle-concurrents
+      version: "2.14"
+    - name: test-stability
+      version: "2.3"
+    - name: postbuildscript
+      version: "3.2.0-550.v88192b_d3e922"
+    - name: github
+      version: "1.37.3"
+    - name: timestamper
+      version: "1.26"
+    - name: openJDK-native-plugin
+      version: "1.7"
+    basePlugins:
+    - name: kubernetes
+      version: "4029.v5712230ccb_f8"
+    - name: workflow-job
+      version: "1342.v046651d5b_dfe"
+    - name: workflow-aggregator
+      version: "596.v8c21c963d92d"
+    - name: git
+      version: "5.2.0"
+    - name: job-dsl
+      version: "1.84"
+    - name: configuration-as-code
+      version: "1700.v6f448841296e"
+    - name: kubernetes-credentials-provider
+      version: "1.225.v14f9e6b_28f53"
+    disableCSRFProtection: false
+    containers:
+      - name: jenkins-master
+        image: jenkins/jenkins:2.414.1-lts-jdk11
+        # env:
+        #   - name: JAVA_OPTS
+        #     value: "-Dhudson.cli.CLI.HTTPS=true -Djenkins.websocket=false"
+        imagePullPolicy: Always
+        livenessProbe:
+          failureThreshold: 12
+          httpGet:
+            path: /login
+            port: http
+            scheme: HTTP
+          initialDelaySeconds: 100
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 5
+        readinessProbe:
+          failureThreshold: 10
+          httpGet:
+            path: /login
+            port: http
+            scheme: HTTP
+          initialDelaySeconds: 80
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 1
+        resources:
+          limits:
+            cpu: 1500m
+            memory: 3Gi
+          requests:
+            cpu: 500m
+            memory: 500Mi
+    securityContext:
+      fsGroup: 1000
+      runAsUser: 1000
+  seedJobs:
+  - id: jenkins-operator
+    targets: ".jenkins/job/DslJob.jenkins" #"cicd/jobs/k8s.jenkins" 
+    description: "Casandra Repo"
+    repositoryBranch: infracloud/cassandra-5.0 #master #infracloud/cassandra-5.0 #
+    repositoryUrl: https://github.com/infracloudio/cassandra.git
```

Comment:
can we parameterise these please. without local cassandra_dir to k8s reverse filesystem sharing, be able to specify these parameters on the run-ci.sh command line is critical functionality

Reply:
Done